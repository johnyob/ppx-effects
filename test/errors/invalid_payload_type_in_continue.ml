let () =
  match%continue k () with
  | [%effect ()] -> ()
;;
