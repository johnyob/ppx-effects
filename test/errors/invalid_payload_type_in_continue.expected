File "invalid_payload_type_in_continue.ml", line 3, characters 4-16:
Error: ppx_effects: invalid node `[%effect <expr>]` used as a pattern.
       Hint: did you mean to use `[%effect? <pattern>]` instead?
